### PR TITLE
Implement header with svg logo and buttons

### PR DESCRIPTION
**Pull request by nmauersberg**
This PR implements a header with buttons that were mocked atm.

It adds the @svgr/webpack package to the playground to import svg files and generates an ts react component from their content.

Tasks:
- [x] Add header like in figma layout
- [x] Use IconButton from components folder
- [x] Ink Logo as component in components folder

### CODE REVIEW

File: packages/playground/src/app/Header/index.tsx [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+    <div className="header">
+      <Logo className="h-16 w-32" data-testid="headerLogo" />
+      <div className="verticalDivider" />
+      <div className={'flex p-3.5 w-full'}>
+        <ButtonWithIcon
+          label="Compile"
+          Icon={CompileIcon}
+          testId={'buttonIcon'}
+          onClick={() => {
+            alert('Compile!');
```
Comment (thought2) [inline packages/playground/src/app/Header/index.tsx:22]: yesterday the async action PR was merged. We could use the real API call here already.

File: packages/playground/src/app/Header/index.tsx [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+        <ButtonWithIcon
+          label="Settings"
+          Icon={SettingsIcon}
+          testId={'buttonIcon'}
+          onClick={() => {
+            alert('Settings!');
+          }}
+        />
+
+        <div className="flex-grow"></div>
```
Comment (thought2) [inline packages/playground/src/app/Header/index.tsx:42]: `<div className="flex-grow"/>`

**Review by thought2 — CHANGES_REQUESTED** [review]
lgtm. just small things + the real async backend call.

File: packages/playground/src/app/Header/index.tsx (hunk 1 above)
Comment (nmauersberg) [reply packages/playground/src/app/Header/index.tsx:22]: Oh cool, yes, I'll do that.

**Review by achimcc — APPROVED** [review]

**Review by thought2 — APPROVED** [review]

**Pull request merged**